apps/src/templates/DialogComponent.jsx

```diff
@@ -45,7 +47,7 @@ var Dialog = React.createClass({
         <div className="modal-backdrop in" onClick={this.closeDialog}></div>
         <div tabIndex="-1" className="modal dash_modal in" ref="dialog" onKeyDown={this.closeOnEscape}>
           <div className="modal-body dash_modal_body">
-            <div id="x-close" className="x-close" onClick={this.closeDialog}></div>
+            {this.props.uncloseable || <div id="x-close" className="x-close" onClick={this.closeDialog}></div>}
```

Comment:
I would prefer to see this written as `!this.props.uncloseable && <div .../>` though I'm not sure I can fully justify why. I think maybe because it makes more sense for me to have an expression result in false and display nothing than it does to result in true and display nothing.